README.md

```diff
@@ -15,3 +13,309 @@ To begin, clone this repository. `cd` into your chosen folder, and run the follo
 JavaScript in this tool is linted with [ESLint](http://eslint.org/) according to our code [syntax and style standards](https://github.com/mobify/mobify-code-style) here at Mobify.
 
 Linting may be run with the `grunt lint` command. Code is also linted automatically on [CircleCI](https://circleci.com/).
+
+## Assertions
+
+#### elementsCount(selector, expected, message, callback)
+
+The `elementsCount` assertion checks if the given selector is present the number of times that is expected for that selector to appear.
+
+Parameter Name | Parameter Type | Description
+--- | --- | ---
```

Comment:
This is a non-blocking thing, but with markdown tables like this whitespace and extra characters for the horizontal rules are ignored. This means you can space things out so that a maintainer can still get a good impression of the table while just viewing the source, like this (check this comment's source):

       Key        |        Value
----------------- | -------------------
       key1       |        value1
       key2       |        value2